BingSearchAPIClient/BingSearchResponse.cs

```diff
@@ -1,14 +1,10 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.Collections.Generic;
 
 namespace BingSearchClient
 {
     public class BingSearchResponse
     {
-        public List<Value> Values { get; set; }
+        public List<Value> Value { get; set; }
```

Comment:
IEnumerable<>, rather than List<>, as you can alter the internal concrete implementation without affecting consumers... ;)